jax/_src/ffi.py

```diff
@@ -15,9 +15,48 @@
 from __future__ import annotations
 
 import os
+import ctypes
 
 from jax._src.lib import jaxlib
 
+# Use Python stable C API function to construct a PyCapsule.
```

Comment:
Nit: this comment should probably be in the function body rather than before it.

Part of it seems to belong in the docstring, and part, I think can be removed.

Reply:
Agreed, I made the comment more concise and moved it into the function body. Didn't see anything that would fit well in the docstring.